extensions/wikia/CommunityPage/templates/helpModule.mustache

```diff
@@ -0,0 +1,4 @@
+<div class="module helpModule">
+	<h2>{{helpModule.title}}</h2>
+	<a data-tracking="help-page-link" href="{{helpModule.link}}">{{helpModule.linkText}} &#8594;</a>
```

Comment:
From what I see we don't watch for all link clicks on community page but we set this up per module, so we need to set this for help module as well.